salt/modules/mysql.py

```diff
@@ -1080,6 +1080,7 @@ def user_exists(user,
                 host='localhost',
                 password=None,
                 password_hash=None,
+                password_column='Password',
```

Comment:
Can you move this line to the end of the `kwarg` statements? We want to avoid an API change here.